libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -147,17 +150,21 @@ export const HealthInsuranceForm: Form = buildForm({
               width: 'half',
               disabled: true,
               defaultValue: (application: Application) =>
-                (application.externalData.nationalRegistry?.data as {
+                ((application.externalData.nationalRegistry?.data as {
+                  address?: Address
+                }).address as {
                   postalCode?: string
-                })?.postalCode,
+                })?.postalCode || '',
```

Comment:
Should we have this: `|| ''` or was it accidentally comitted?

Reply:
If we don't have this we get an error `Cannot convert undefined or null to object` so this is for us when we are testing so that we can move forward in the form. `|| ''` is only after postal code since it is the only one we don't get with our test user